packages/strapi-plugin-upload/admin/src/components/FiltersCard/Wrapper.js

```diff
@@ -0,0 +1,5 @@
+import styled from 'styled-components';
+
+const Wrapper = styled.div``;
```

Comment:
Is this component needed?

Reply:
Yes, because the filters compos will be updated in this draft PR #5333